Source/Core/Core/Boot/Boot.cpp

```diff
@@ -226,7 +226,11 @@ bool CBoot::BootUp()
 	NOTICE_LOG(BOOT, "Booting %s", _StartupPara.m_strFilename.c_str());
 
 	g_symbolDB.Clear();
-	VideoInterface::Preset(_StartupPara.bNTSC);
+
+	// PAL Wii uses NTSC framerate and linecount in 60Hz/480i and 60Hz/480p modes
+	const bool bPAL60 = _StartupPara.bWii && (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60") || SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.PGS"));
```

Comment:
Yeah that was my idea there, blocking the on-console illegal combination of 576p while assuming the Progressive option had higher priority. I suppose we could just allow it, but I dunno if it doesn't lead to odd behavior if it can't actually occur on console.